src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -603,7 +604,8 @@ protected void addTranslations()
 		addStoreTooltip(MSItems.CARVING_TOOL, "You gotta BE the marble!");
 		add(MSItems.CRUMPLY_HAT, "Crumply Hat");
 		add(MSItems.STONE_EYEBALLS, "Stone Eyeballs");
-		add(MSItems.STONE_SLAB, "Stone Slab");
+		add(MSItems.STONE_SLAB, "Stone Tablet");
+		addExtra(MSItems.STONE_SLAB, "carved", TextFormatting.GRAY+"Looks like something has been carved into it!");
```

Comment:
I'd suggest setting the text formatting to the text component in `StoneTabletItem.addInformation()` instead of here